apps/explorer/src/pages/transaction-result/programmable-transaction-view/Transaction.tsx

```diff
@@ -1,53 +1,60 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+import { useCopyToClipboard } from '@mysten/core';
+import { Copy12 } from '@mysten/icons';
 import {
     type MoveCallSuiTransaction,
     type SuiArgument,
     type SuiMovePackage,
 } from '@mysten/sui.js';
 import { type ReactNode } from 'react';
+import { toast } from 'react-hot-toast';
 
 import { flattenSuiArguments } from './utils';
 
 import { ErrorBoundary } from '~/components/error-boundary/ErrorBoundary';
 import { ObjectLink } from '~/ui/InternalLink';
+import { Text } from '~/ui/Text';
+
+function CopyClipboard({ copyText }: { copyText: string }) {
+    const copyToClipBoard = useCopyToClipboard(copyText, () =>
+        toast.success('Copied!')
+    );
+
+    return (
+        <Copy12
+            className="h-3 w-3 cursor-pointer text-gray-45 hover:text-steel"
+            onClick={copyToClipBoard}
+        />
+    );
+}
 
 export interface TransactionProps<T> {
     type: string;
     data: T;
 }
 
-function TransactionContent({
-    type,
-    children,
-}: {
-    type: string;
-    children?: ReactNode;
-}) {
+function TransactionContent({ children }: { children?: ReactNode }) {
     return (
-        <>
-            <div className="break-all text-heading6 font-semibold text-steel-darker">
-                {type}
-            </div>
-            {children && (
-                <div className="text-bodyMedium pt-2 font-medium text-steel-dark">
-                    {children}
-                </div>
-            )}
-        </>
+        <Text variant="pBody/normal" color="steel-dark">
+            {children}
+        </Text>
     );
 }
 
 function ArrayArgument({
-    type,
     data,
 }: TransactionProps<(SuiArgument | SuiArgument[])[] | undefined>) {
+    const renderData = data ? `(${flattenSuiArguments(data)})` : '';
```

Comment:
Just define this inline

Reply:
Need this value for `copyToClipboard`

Reply:
Why is this value copyable? This isn't a useful value to copy?

Reply:
![Screenshot 2023-04-29 at 1 23 30 PM](https://user-images.githubusercontent.com/127577476/235322801-d88ca0e6-b7e3-4fd8-926e-8243f9531a2d.png)

I am following this Figma. @mystie711 I do we want to be able to clipboard-copy these?